### ISSUE TITLE
Feature Request: View by Album Artist

### ISSUE THREAD

**Issue by ibleedbinari** [naman14/Timber#130]
On albums created by a collaboration of artists, the album gets split into sometime 5-10 pieces for each artist. This is very frustrating when trying to listen to one Album from start to finish which is made my multiple Artists. A possible solution is to allow using the "Album Artist" ID3 tag as priority over "Artist" when it exists.

**Comment by ibleedbinari** [comment]
Another possible solution (both should be implemented) is to not separate the Albums by their Artist in the "Albums" tab. The is what VLC for Android does, and it works great.

**Comment by mihawk90** [comment]
> to not separate the Albums by their Artist in the "Albums" tab. 

Imo that should be the case anyway, an album is an album, no matter the artists 

That aside, I found that album artist is actually already used when no artist is present in the tags (as a fallback), it should just be the other way round (and/or optional).
I have some test files which I just added an album artist for testing with various players (it's amazing how few players actually have album artist sorting), and that shows up. Another set of files had album artist and artist, where all artists show up in the artist tab.

**Comment by rivaldi8** [comment]
@mihawk90 It doesn't work :( It says the column doesn't exists.

**Comment by vanyasem** [comment]
@rivaldi8 of course, it doesn't work, there is no such column in [MediaStore.Audio.ArtistColumns](https://developer.android.com/reference/android/provider/MediaStore.Audio.ArtistColumns.html). It's an issue in Android since 1.0 (API 1). The only solution would be to implement custom music scanning with support for Album Artist tag. This is the way it's implemented in Google Play Music.

**Comment by rivaldi8** [comment]
@vanyasem Yes, I found out later. Implementing a custom music scanning would be quite a lot of work. Are you sure about Google Play Music? I get the same list as with Timber.

**Comment by vanyasem** [comment]
@rivaldi8 yep, Google Play Music gives expected results

## Timber Artists:
![Timber Artists](https://user-images.githubusercontent.com/13350458/30523326-4b928be2-9be8-11e7-955f-f0769ef88a61.png)

### Timber Artist Page:
![Timber Artist Page](https://user-images.githubusercontent.com/13350458/30523310-1d24eb06-9be8-11e7-85b0-42bb2acd3c34.jpg)
_(note missing tracks)_


## Google Music Artists:
![Google Music Artists](https://user-images.githubusercontent.com/13350458/30523322-43215e52-9be8-11e7-9be7-cf6dcc136607.png)

### Google Music Artist Page:
![Google Music Artist Page](https://user-images.githubusercontent.com/13350458/30523315-33e39892-9be8-11e7-8f85-fa5966957044.png)
_(all tracks are displayed)_

**Comment by neubland** [comment]
Shuttle music player has found a solution and I think they don't have a custom music scanner. They're not open source, but maybe one could ask? Also it would be possible to use MediaStore for scanning but building an own database from there with separate ID3-tag-reading. This would facilitate better cover art recognition too.

**Comment by vanyasem** [comment]
https://github.com/timusus/Shuttle is it really not open source lol? you are really misteken @neubland

**Comment by vanyasem** [comment]
i compiled it myself from source and have been using it for a few months now
